### PR TITLE
Improve xamarin-jdk

### PR DESCRIPTION
**Pull request by scribblemaniac**

### CODE REVIEW

File: Casks/xamarin-jdk.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   version '1.7.0_71'
   sha256 '70a18547b529a111c4e5cf133532082e142908819b0d61e273c21dee86fcc87a'
 
-  url 'https://download.xamarin.com/Installer/MonoForAndroid/jdk-7u71-macosx-x64.dmg'
+  url "https://download.xamarin.com/Installer/MonoForAndroid/jdk-#{version.sub(%r{^\d+\.(\d+).*?_(.*)$},'\1u\2')-macosx-x64.dmg"
```
Comment (jawshooah) [inline Casks/xamarin-jdk.rb]: Looks like you forgot the closing `}` in your interpolation block.

File: Casks/xamarin-jdk.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@
   homepage 'https://xamarin.com/platform'
   license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
 
-  pkg 'JDK 7 Update 71.pkg'
+  pkg "JDK #{version.split('.')[1]} Update #{version.split('_')[1]}.pkg"
 
-  uninstall :pkgutil => 'com.oracle.jdk7u71'
+  uninstall :pkgutil => "com.oracle.jdk#{version.sub(%r{^\d+\.(\d+).*?_(.*)$},'\1u\2')"
```
Comment (jawshooah) [inline Casks/xamarin-jdk.rb]: Here as well.

**Comment by scribblemaniac** [comment]
@jawshooah Good catch, fixed.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 06f1489fa6654f162986bd40bbee3f0171dee345 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**